patches/vstest/0009-Suppress-Mono-check-during-build.-See-https-github.c.patch

```diff
@@ -0,0 +1,44 @@
+From 43d6b05323e27cadc40294f4a2591d91a8ab1d89 Mon Sep 17 00:00:00 2001
+From: Eric Erhardt <eric.erhardt@microsoft.com>
+Date: Fri, 29 Sep 2017 10:46:12 -0500
+Subject: [PATCH] Suppress Mono check during build. See
+ https://github.com/Microsoft/vstest/issues/1153
+
+---
+ scripts/build.sh | 6 +++++-
+ 1 file changed, 5 insertions(+), 1 deletion(-)
+
+diff --git a/scripts/build.sh b/scripts/build.sh
+index 342be5f..3057831 100755
+--- a/scripts/build.sh
++++ b/scripts/build.sh
+@@ -22,6 +22,7 @@ FAIL_FAST=false
+ DISABLE_LOCALIZED_BUILD=false
+ CI_BUILD=false
+ VERBOSE=false
++CHECK_MONO=true
+ PROJECT_NAME_PATTERNS=
+ 
+ while [ $# -gt 0 ]; do
+@@ -62,6 +63,9 @@ while [ $# -gt 0 ]; do
+         -verbose)
+             VERBOSE=true
+             ;;
++        -nomono)
++            CHECK_MONO=false
++            ;;
+         *)
+             break
+             ;;
+@@ -102,7 +106,7 @@ TPB_Version=$(test -z $VERSION_SUFFIX && echo $VERSION || echo $VERSION-$VERSION
+ TPB_CIBuild=$CI_BUILD
+ TPB_LocalizedBuild=$DISABLE_LOCALIZED_BUILD
+ TPB_Verbose=$VERBOSE
+-TPB_HasMono=$(command -v mono > /dev/null && echo true || echo false)
++TPB_HasMono=$CHECK_MONO && $(command -v mono > /dev/null && echo true || echo false)
```

Comment:
It looks to me like this should be:

```shell
TPB_HasMono=$($CHECK_MONO && command -v mono > /dev/null && echo true || echo false)
```

Otherwise the `&&` is after the full command `TPB_HasMono=$CHECK_MONO`, so the rest of the line doesn't affect `TPB_HasMono`.

Reply:
Good call.  Fixing